adapter-rest-contract/src/main/resources/io/github/fileanalysissuite/adapters/rest/contract/swagger.yaml

```diff
@@ -230,8 +230,6 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/RepositoryFile'
-      required:
-        - files
```

Comment:
Why are we removing this?

Reply:
Because you might have just retries or failures. I was trying to use oneOf but it generated an invalid c#.

Reply:
But isn't this the request?

Reply:
Obviously, this was a wrong place, reverted